interview-questions.md

```diff
@@ -8,44 +8,45 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a Ruby class?
 
-Your answer:
+Your answer: Within the body of a method, it calls to super acts just like a call to that original method. The search for a method body starts in the superclass of the object that was found to contain the original method.
 
 Researched answer:
+You don't have to type super(from, to) – Ruby will assume that by default. But in the definition ( def initialize ) of the constructor you have to define both parameters, since Ruby doesn't – and shouldn't!
 
 2. What is a gem?
 
-Your answer:
+Your answer: GEM (for Graphics Environment Manager) is an operating environment released by Digital Research (DRI) in 1985 for use with the DOS operating system on Intel 8088 and Motorola 68000 microprocessors.
 
-Researched answer:
+Researched answer: They are simply open source libraries that contain Ruby code and are packaged with a little extra data . Using a gem allows a programmer to use the code within the gem in their own program, without explicitly inserting that code.
```

Comment:
This is a good start! You can use examples and I would consider maybe finding one or two that catch your attention and thinking about how you could implement them in a project.